cmake/ConfigChemps2.cmake

```diff
@@ -9,24 +9,25 @@ if(NOT CHEMPS2_FOUND)
     message(STATUS "CheMPS2 not found. The pre-packaged version will be built.")
 
     find_package (HDF5 QUIET)
-    find_package (GSL QUIET)
-    if (NOT GSL_FOUND OR NOT HDF5_FOUND)
-        message(FATAL_ERROR "No GSL or HDF5, no CheMPS2. Build against existing with -DCHEMPS2_ROOT=$CONDAENV or skip with -DENABLE_CHEMPS2=OFF")
+    if (NOT HDF5_FOUND)
+        message(FATAL_ERROR "No HDF5 or CheMPS2 found. Build against an existing CheMPS2 with -DCHEMPS2_ROOT=$CONDAENV or skip with -DENABLE_CHEMPS2=OFF")
     endif()
 
     set(CUSTOM_CHEMPS2_LOCATION ${PROJECT_BINARY_DIR}/interfaces/chemps2)
     include(ExternalProject)
     ExternalProject_Add(interface_chemps2
         PREFIX ${CUSTOM_CHEMPS2_LOCATION}
         GIT_REPOSITORY https://github.com/SebWouters/CheMPS2
-        GIT_TAG v1.5
-        CMAKE_ARGS -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+        GIT_TAG v1.7
+        CMAKE_ARGS 
+                   -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+                   -DEXTRA_C_FLAGS=${CMAKE_EXTRA_C_FLAGS}
+                   -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+                   -DEXTRA_CXX_FLAGS=${CMAKE_EXTRA_CXX_FLAGS}
                    -DSTATIC_ONLY=ON
                    -DENABLE_TESTS=OFF
                    -DENABLE_GENERIC=${ENABLE_STATIC_LINKING}
                    -DENABLE_XHOST=${ENABLE_XHOST}
-                   -DLAPACK_LIBRARIES=${BLAS_LIBRARIES}
-                   -DGSL_LIBRARIES=${GSL_LIBRARIES}
```

Comment:
They now appear on line 42. The current setup was detecting them incorrectly so I switched to an ambit like example which appeared to work better. If there is a current preferred way that works I'm all for it